src/test/java/com/easyrest/tests/administrator/AdministratorTest.java

```diff
@@ -30,18 +31,24 @@ public void signIn() {
 
     @Test
     public void acceptWaitingForConfirmOrder() {
+        OrderDao orderDao = new OrderDao();
+        int insertedOrderId = orderDao.saveWaitingForConfirmOrder(email);
         Integer ordersCountBeforeAccepting = administrator.seeWaitingForConfirmOrdersCount();
         administrator.acceptOrder();
         Integer ordersCountAfterAccepting = administrator.seeWaitingForConfirmOrdersCount();
+        orderDao.deleteOrder(insertedOrderId);
 
         Assert.assertEquals(ordersCountAfterAccepting, ordersCountBeforeAccepting - 1, "Order is not accepted");
     }
 
     @Test
     public void assignWaiterForAcceptedOrder() {
+        OrderDao orderDao = new OrderDao();
+        int insertedOrderId = orderDao.saveAcceptedOrder(email);
         Integer ordersCountBeforeAssigning = administrator.seeAcceptedOrdersCount();
         administrator.assignWaiter();
         Integer ordersCountAfterAssigning = administrator.seeAcceptedOrdersCount();
+        orderDao.deleteOrder(insertedOrderId);
```

Comment:
why do you delete an order at this point and not in the teardown method?

Reply:
Fixed. I have moved the order deleting to the teardown method.